opendebates/templates/opendebates/top_archive.html

```diff
@@ -0,0 +1,53 @@
+{% extends "opendebates/base.html" %}
+{% block content %}
+
+{% load i18n l10n %}
+
+<div class="row">
+  <div class="col-sm-8 col-sm-offset-2">
+    {{ category.title|safe }}
+    <div class="row idea-list">
+      <div class="winners">
+        {% for idea in submissions %}
+        <div id="i{{ idea.submission.id }}"
+             data-idea-id="{{ idea.submission.id }}"
+             class="big-idea clearfix">
+          <div class="votes col-md-3">
+            <div class="vote-tally text-center">
+              <span
+                 class="vote-tally-number">{{ idea.votes|localize }}</span>
+            </div>
+            <div class="vote-bottom center-block center-text">
+              <br>
```

Comment:
`<p>` picks up some extra styles that we don't want (https://cl.ly/0g2a1Z0e0u0q, vs the desired https://cl.ly/3F1t0D0f0G0d) and with upcoming events looming I'm reluctant to start making any CSS changes, since this markup is so closely shared with the existing `snippets/idea.html` code.